app/templates/views/service-join-request-user-already-joined.html

```diff
@@ -0,0 +1,18 @@
+{% extends "withnav_template.html" %}
+{% from "components/page-header.html" import page_header %}
+
+{% block service_page_title %}
+  This person is already a team member
+{% endblock %}
+
+{% block maincolumn_content %}
+
+  {{ page_header('This person is already a team member') }}
+
+  <p class="govuk-body">
+    {{ user_to_invite.name }} is already member of '{{ current_service.name }}'.
```

Comment:
Use smart/curly quotes

```suggestion
    {{ user_to_invite.name }} is already member of ‘{{ current_service.name }}’.
```